modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,49 +5,45 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+== SSO levels
 
-[NOTE]
-====
-Accessing Aura with SSO requires:
- 
-* Authorization Code Flow with PKCE.
-* A publicly accessible Identity Provider (IdP) server.
-====
+Aura admins can configure organization and tenant level SSO:
 
-== Console SSO
+. *Organization level SSO*: This configuration applies to all tenants and new database instances created within your organization.
 
-Console SSO allows you to log in to the Aura Console using company IdP credentials and grants link:{neo4j-docs-base-uri}/cypher-manual/current/administration/access-control/built-in-roles#access-control-built-in-roles-public[Public Access privileges] to all instances in the tenant.
+. *Tenant level SSO*:  SSO configured at the tenant level affects all new database instances created within that tenant. It ensures users logging in with SSO have access to the authorized database instances within the tenant.
 
-The following OpenID Connect (OIDC) certified Identity Providers (IdPs) are currently supported for Console-level Authentication:
+== Login methods
 
-* Microsoft Entra ID
+Admins can configure a combination of one or more of the following login methods:
+
+* Email/password
 * Okta
+* Microsoft Entra ID
+* Google Sign-in
```

Comment:
So these login methods are different for each sso level. 

Org sso supports:
- email/pw
- Google Social Login (not the customers own google workspace sso)
- Microsoft Entra ID
- Okta

Where as instance sso supports:
- user/pw
- Microsoft Entra ID
- Okta

However, at the instance level, you cannot disable user/pw but in org sso you can disable email/pw and google social login so long as you have at least one other custom sso provider configured

Reply:
What's the link to Google's landing page for the Google Social Login product?

Reply:
I have no idea, couldn't find anything via a quick google search. Is there some specific information around the login you are looking for?

Reply:
I'd like to make sure I have the exact name for the Google feature - as far as I can see, there isn't a Google product called "Google Social Login"

Would it be correct to call it "SSO with Google" (Google Support call it that here and it sounds nice: https://support.google.com/a/answer/12032922?hl=en)

Reply:
Sounds good for me, but probably something best to check with Aman, Johnny, and Elfreda if theyre happy with that. I know we've been calling Google Social Login internally for awhile. Otherwise SSO with Google sounds good.

Reply:
We're going to go with "Google SSO"